pyanaconda/modules/storage/partitioning/interactive/utils.py

```diff
@@ -227,29 +227,15 @@ def create_new_root(storage, boot_drive):
         boot_drive=boot_drive
     )
 
-    bootloader_devices = collect_bootloader_devices(
-        storage=storage,
-        boot_drive=boot_drive
-    )
-
-    swaps = [
-        d for d in devices
-        if d.format.type == "swap"
-    ]
-
     mounts = {
         d.format.mountpoint: d for d in devices
         if getattr(d.format, "mountpoint", None)
     }
 
-    for device in devices:
-        if device in bootloader_devices:
-            mounts[device.format.name] = device
```

Comment:
I have a bad feeling about removing this part. We don't really need it?

Reply:
Well, this is exactly that code that adds bootloader devices to a dictionary of mount points, so if you have two biosboot devices, you will end up with only one of them. Without this change, the rest of it is pointless.